src/TorchSharp/TorchVision/Video.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Threading.Tasks;
+
+using static TorchSharp.torch;
+
+namespace TorchSharp.torchvision
+{
+    public static partial class io
+    {
+        public interface IVideoer
```

Comment:
Based on your comment, let's leave video processing out until someone proposes it.